pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -249,16 +247,13 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	}
 
 	// As of Nautilus Ceph auto-tunes its osd_memory_target on the fly so we don't need to force it
-	if !c.clusterInfo.CephVersion.IsAtLeastNautilus() && !c.resources.Limits.Memory().IsZero() {
+	if !c.resources.Limits.Memory().IsZero() {
```

Comment:
@leseb This block should've been removed, right? The old code was only applying it if the version was below Nautilus.

Reply:
Good catch, let me fix this.

Reply:
See: https://github.com/rook/rook/pull/5529

Reply:
Expect a backport soon